### PR TITLE
makes the nuke op leader getting the war declaration a config

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

what the title says

## Why It's Good For The Game

seems useful if admins would like to disable things like war ops where everyone becomes a ravaging monki on manuel or some other servers without deleting it everywhere or throwing the whole gamemode out of rotation

## Changelog
:cl:
config: the nuke op leader having access to the war declaration item is now a config
/:cl:

### CODE REVIEW

**Comment by necromanceranne** [comment]
I love war ops but like I'm against more configs.

**Comment by optimumtact** [comment]
remove war ops

**Comment by BadSS13Player** [comment]
I thought people hated stealthops, not warops.

**Comment by knacker48** [comment]
People are contrarians that hate everything

**Comment by Spepbuthewranglesgithub** [comment]
more options is almost always a good thing, get this thing to market.

**Comment by Tlaltecuhtli** [comment]
normal ops sucks because once ops blow up brig its just soldiers vs civilians congo simulation 

war ops sucks for the 20 min delay where you wait sitting on your contract issued ar 15 to then run around the station finding out the ops landed to the  other side and have all died

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Update `config/game_options.txt`

File: code/modules/antagonists/nukeop/nukeop.dm [hunk 1]
```diff
@@ -179,7 +179,15 @@
 	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ops.ogg',100,0)
 	to_chat(owner, "<B>You are the Syndicate [title] for this mission. You are responsible for the distribution of telecrystals and your ID is the only one who can open the launch bay doors.</B>")
 	to_chat(owner, "<B>If you feel you are not up to this task, give your ID to another operative.</B>")
-	to_chat(owner, "<B>In your hand you will find a special item capable of triggering a greater challenge for your team. Examine it carefully and consult with your fellow operatives before activating it.</B>")
+	if(!CONFIG_GET(flag/disable_warops))
+		to_chat(owner, "<B>In your hand you will find a special item capable of triggering a greater challenge for your team. Examine it carefully and consult with your fellow operatives before activating it.</B>")
+		var/obj/item/nuclear_challenge/dukinuki = new
+		var/mob/living/carbon/human/H = owner.current
+		if(!istype(H))
+			dukinuki.forceMove(get_turf(H))
```
Comment (Cyberboss) [inline code/modules/antagonists/nukeop/nukeop.dm]:
```suggestion
			dukinuki.forceMove(H.drop_location())
```

File: code/modules/antagonists/nukeop/nukeop.dm [hunk 2]
```diff
@@ -179,7 +179,15 @@
...
+		else
+			H.put_in_hands(dukinuki, TRUE)
+			H.update_icons()
```
Comment (Cyberboss) [inline code/modules/antagonists/nukeop/nukeop.dm]:
```suggestion
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/antagonists/nukeop/nukeop.dm [hunk 3]
```diff
@@ -179,7 +180,14 @@
 	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ops.ogg',100,0)
 	to_chat(owner, "<B>You are the Syndicate [title] for this mission. You are responsible for the distribution of telecrystals and your ID is the only one who can open the launch bay doors.</B>")
 	to_chat(owner, "<B>If you feel you are not up to this task, give your ID to another operative.</B>")
-	to_chat(owner, "<B>In your hand you will find a special item capable of triggering a greater challenge for your team. Examine it carefully and consult with your fellow operatives before activating it.</B>")
+	if(!CONFIG_GET(flag/disable_warops))
+		to_chat(owner, "<B>In your hand you will find a special item capable of triggering a greater challenge for your team. Examine it carefully and consult with your fellow operatives before activating it.</B>")
+		challengeitem = new
```
Comment (Cyberboss) [inline code/modules/antagonists/nukeop/nukeop.dm]: Pretty sure this is not going to create the right type

Comment (ExcessiveUseOfCobblestone) [reply code/modules/antagonists/nukeop/nukeop.dm]: i think you're looking for

```DM
var/.../dukienukie = new challengeitem
```

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**